### DISCUSSION TITLE
无限长序列、惰性计算与C++(1.5) – ChloroBlog

### DISCUSSION THREAD

**Discussion by Chlorie** [Chlorie/ChloroBlog#20] [Announcements]
https://chlorie.github.io/ChloroBlog/posts/2019-08-15/range-1-extra.html 

 能鸽善鹉者的领域

**Comment by c0nstexpr** [comment]
```开始逐渐看不懂.jpg```

**Comment by Chlorie** [comment]
> @BlurringShadow
> ```开始逐渐看不懂.jpg```

其实这次还都是挺简单的东西啊（（
我马上去写```zip```的实现（
